src/components/core/pivotTable/aggregationsMenuHelper.ts

```diff
@@ -81,5 +86,6 @@ export default {
     getTotalsForAttributeHeader,
     getTotalsForMeasureHeader,
     getHeaderMeasureLocalIdentifiers,
-    isTotalEnabledForAttribute
+    isTotalEnabledForAttribute,
+    getTotalTypesAppliedOnMeasures: getTotalTypesAppliedOnAllMeasures
```

Comment:
You forgot to rename this manually (side effect of IDEA refactor).

Reply:
will wix